plugins/connection/incus.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# Based on lxd.py (c) 2016, Matt Clay <matt@mystile.com>
+# (c) 2023, Stephane Graber <stgraber@stgraber.org>
+# Copyright (c) 2023 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author: Stéphane Graber <stgraber@stgraber.org>
+    connection: incus
+    short_description: Run tasks in Incus instances via the Incus CLI
+    description:
+        - Run commands or put/fetch files to an existing Incus instance using Incus CLI
+    version_added: "2.0"
```

Comment:
The next minor release of community.general is 8.2.0:
```suggestion
    version_added: "8.2.0"
```